cpp/Doxyfile.in

```diff
@@ -737,7 +737,7 @@ WARN_NO_PARAMDOC       = YES
 # a warning is encountered.
 # The default value is: NO.
 
-WARN_AS_ERROR          = YES
+WARN_AS_ERROR          = NO
```

Comment:
```suggestion
WARN_AS_ERROR          = YES
```

Can we revert this? I think this is good to keep enabled.

In our local debugging session, I was just using it temporarily to see if we could get a successful build.

If we run into any errors as result of reverting this, we can solicit help from Dante.

Reply:
changed it, had to add a doctring to a `hpp` file to avoid one warning, but everything compiles now locally